### PR TITLE
Remove CentOS8 support

### PR DESCRIPTION
**Pull request by jt-edb**

### CODE REVIEW

File: edbdeploy/data/vmware-wkstn/r8-EDB-RA-1 [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "provider": "vmware",
+    "shared_folders": [
+      {
+        "host_path": ".",
+        "share_name": "mech"
+      }
+    ],
+    "url": "https://app.vagrantup.com/rockylinux/boxes/8/versions/4.0.0/providers/virtualbox.box"
+  }
+}
```
Comment (vincentp7212) [inline edbdeploy/data/vmware-wkstn/r8-EDB-RA-1:41]: The Official Rockylinux does not support VMWare when it comes to Vagrant Boxes. I found another image that does. This is the reformatted file.

{
  "pem1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "pem1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "barman1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "barman1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "primary1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "primary1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  }
}

File: edbdeploy/data/vmware-wkstn/r8-EDB-RA-2 [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    "provider": "vmware",
+    "shared_folders": [
+      {
+        "host_path": ".",
+        "share_name": "mech"
+      }
+    ],
+    "url": "https://app.vagrantup.com/rockylinux/boxes/8/versions/4.0.0/providers/virtualbox.box"
+  }
+}
```
Comment (vincentp7212) [inline edbdeploy/data/vmware-wkstn/r8-EDB-RA-2:67]: The Official Rockylinux does not support VMWare when it comes to Vagrant Boxes. I found another image that does. This is the reformatted file.

{
  "barman1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "barman1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "pem1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "pem1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "primary1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "primary1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "primary2": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "primary2",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "primary3": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "primary3",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  }
}

File: edbdeploy/data/vmware-wkstn/r8-EDB-RA-3 [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    "provider": "vmware",
+    "shared_folders": [
+      {
+        "host_path": ".",
+        "share_name": "mech"
+      }
+    ],
+    "url": "https://app.vagrantup.com/rockylinux/boxes/8/versions/4.0.0/providers/virtualbox.box"
+  }
+}
```
Comment (vincentp7212) [inline edbdeploy/data/vmware-wkstn/r8-EDB-RA-3:106]: {
  "pem1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "pem1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "barman1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "barman1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "primary1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "primary1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "primary2": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "primary2",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "primary3": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "standby3",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "pooler1": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "pooler1",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "pooler2": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "pooler2",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  },
  "pooler3": {
    "box": "bento/rockylinux-8",
    "box_version": "202112.19.0",
    "name": "pooler3",
    "provider": "vmware",
    "shared_folders": [
      {
        "host_path": ".",
        "share_name": "mech"
      }
    ],
    "url": "https://vagrantcloud.com/bento/boxes/rockylinux-8/versions/202112.19.0/providers/vmware_desktop.box"
  }
}

**Review by vincentp7212 — COMMENTED** [review]
Remove mentions of 'CentOS8' from all READ.ME files (including VMWare and Virtualbox README)

**Comment by jt-edb** [comment]
> Remove mentions of 'CentOS8' from all READ.ME files (including VMWare and Virtualbox README)

Done

**Pull request merged**